netbox/ipam/models/vlans.py

```diff
@@ -114,6 +114,15 @@ def get_next_available_vid(self):
             return available_vids[0]
         return None
 
+    def get_utilization(self):
+        """
+        Return the percentage of utilization for this vlan group.
+        """
+        assigned_vlan = VLAN.objects.filter(group=self.id).count()
```

Comment:
Just to reiterate what we touched on in today's maintainer meeting: We can't call `count()` on each instance as it will result in a database query for each object being returned as part of an API response. Instead, we can annotate the queryset under the view to include the count of assigned VLANs per VLAN range.

Reply:
I have made the changes as per the suggestion